packages/juce-blueprint/template/src/index.js

```diff
@@ -1,5 +1,3 @@
-import "juce-blueprint/cjs/lib/polyfill";
```

Comment:
Once this goes in I can do a follow up PR with a TS project template. We can maybe provide templates for both TS and JS users and have juce-blueprint provide an `init-js` command and a `init-ts` command or just have the `init` command take a `useTS` flag or something. Then the `init.js` script can just switch on the template directory to copy based on that flag, i.e. `template-js` or `template-ts`. We can update the docs accordingly

Reply:
Maybe the set of example apps could be the existing GainPlugin for JS, some other simple plugin for TS like DelayPlugin and then the "ComponentShowcase" app which can be either.

Reply:
:+1: definitely, all sounds great